tests/csv/panel_full.csv

```diff
@@ -0,0 +1,23 @@
+panel,chr,vcf,index
```

Comment:
Has this been tested? Is it GRCh37 or GRCh38?

Reply:
Not yet